docs/core/compatibility/core-libraries/6.0/numeric-format-parsing-handles-higher-precision.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Breaking change: Standard numeric format parsing precision"
+description: Learn about the .NET 6.0 breaking change in core .NET libraries where standard numeric format parsing now handles higher precisions.
+ms.date: 02/26/2021
+---
+# Standard numeric format parsing precision
+
+In .NET 6+, .NET supports precision up to <xref:System.Int32.MaxValue?displayProperty=nameWithType> when formatting numbers as strings using `ToString` and `TryFormat`.
```

Comment:
This seems confusing. Considering MaxValue isn't a number of decimal places, but the highest value an Int32 can represent, it seems odd that it's being referenced. I may not be understanding something though. It may be the way the word precision is used.